pkg/kubectl/cmd/get/get_flags.go

```diff
@@ -167,7 +167,7 @@ func (f *PrintFlags) AddFlags(cmd *cobra.Command) {
 
 	// TODO(juanvallejo): This is deprecated - remove
 	cmd.Flags().BoolP("show-all", "a", true, "When printing, show all resources (default show all pods including terminated one.)")
-	cmd.Flags().MarkDeprecated("show-all", "will be removed in an upcoming release")
+	cmd.Flags().MarkDeprecated("show-all", "will be removed in an upcoming release, Pods of all states are shown by default now")
```

Comment:
I don't think this needs to be updated.

Reply:
@dixudx can you say why? As mentioned in #67895, there is no clear indication of what users are supposed to use instead of `--all`, it isn't obvious that `--all` is not the default behavior.